docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -48,7 +53,6 @@ If the prompt does not appear, enable the extension by doing the following:
 . Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary.
 You should receive an output similar to `149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`. If you receive this output, the kernel extension is enabled.
 
-
 [discrete]
 [[system-extension]]
 == Approve the system extension for the Endgame sensor
```

Comment:
git won't allow me to comment on the next lines. So I just comment here. In line 60, it should be:
`you must enable the system extension during installation on macOS Big Sur (11.0) and later.`

Reply:
Made this change in [ca7dc58](https://github.com/elastic/security-docs/pull/3565/commits/ca7dc5896f91b6541a9c2bdaa8c6b012b531795e).